configuration/pipeline_config.json

```
@@ -0,0 +1,223 @@
+{
+  "PipelineName": "EU-PCVE",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/csap-text-it-token.txt",
+      "ActivationFlowNames": [
+        "csap_eu_pcve_s01e01_activation"
+      ],
+      "SurveyFlowNames": [
+        "csap_demog",
+        "csap_s02_demog",
+        "csap_s03_demog",
+        "csap_s09_demog",
+        "csap_eu_pcve_demog"
+      ],
+      "TestContactUUIDs": [
+          "95c57d00-fd3a-407c-ad15-2b8ba7c2154a",
+          "b0534949-700b-4a5c-a358-e2ab9e239124",
+          "6f251195-5889-437a-a8b5-ad74b8e4eb62",
+          "ed65f87f-4805-42ad-9b66-27f34c81c4ff",
+          "e5b3d475-4e73-41a0-80a2-b39b28aad89c",
+          "856a55d8-9159-4970-bd22-ede3ae5f94e0"
+      ]
+    },
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/csap-2-textit-token.txt",
+      "ActivationFlowNames": [],
+      "SurveyFlowNames": [
+        "csap_s04_demog",
+        "csap_s05_demog",
+        "csap_s06_demog",
+        "csap_s07_demog",
+        "csap_s08_demog",
+        "csap_s08_evaluation"
+      ],
+      "TestContactUUIDs": [
+        "61e74e0b-1b4a-41fe-9e78-712f2a048f4e",
+        "d4a80e64-c46a-492d-9b71-bae5fa54a741",
+        "84b4d75d-4c8e-4403-994c-05c1377700f8",
+        "83ce764b-5c80-4d56-9d0f-84435570141d"
+      ]
+    },
+    {
+      "SourceType": "GCloudBucket",
+      "ActivationFlowURLs": [],
+      "SurveyFlowURLs": [
+        "gs://avf-project-datasets/2020/USAID-IBTCI/csap_s10_demog.jsonl"
+      ]
+    }
+  ],
+  "UuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "ADSS",
+    "UuidPrefix": "avf-phone-uuid-"
+  },
+  "OperationsDashboard": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json"
+  },
+  "SourceKeyRemappings": [
+    {"SourceKey": "avf_phone_id", "PipelineKey": "uid"},
+
+    {"SourceKey": "Rqa_EU_PCVE_S01E01 (Text) - csap_eu_pcve_s01e01_activation", "PipelineKey": "rqa_s01e01_raw", "IsActivationMessage": true},
```

```suggestion
    {"SourceKey": "Rqa_Eu_Pcve_S01E01 (Text) - csap_eu_pcve_s01e01_activation", "PipelineKey": "rqa_s01e01_raw", "IsActivationMessage": true},
```

(rapid pro mangles the case of variable names, so it should look like this instead, which you would have seen if you had tested)